_scripts/deploy.sh

```diff
@@ -5,7 +5,7 @@
 
 cd "$(dirname "$0")" || exit 1
 
-export IMAGE_PREFIX=deis VERSION=2.0.0-alpha
+export IMAGE_PREFIX=deis VERSION=canary
```

Comment:
I like that idea. I'll probably be using this convention across all Deis components if we agree this is where we want to go re: HEAD version.